frontend/app/src/i18n/en.json

```diff
@@ -444,6 +444,7 @@
     "hotGroups": "Hot groups",
     "hours": "Hours",
     "howToBuyToken": "I need help buying {token}",
+    "identityMigrationMessage": "Please wait while you account is upgraded to our new system which will support many more sign-in options going forwards. It will also allow us to provide (coming soon) an optional recovery email in case you lose access to your device.",
```

Comment:
This text can be improved

Reply:
Please wait while your account is upgraded to our new system which will support many more sign-in options going forwards. It will also allow you to provide (coming soon) an optional recovery email in case you lose access to your device.

Reply:
Please wait while you account is upgraded to our new system. This will support many more sign-in options going forwards including optional recovery email in case you lose access to your device.